docs/manual/fault_domains.md

```diff
@@ -29,6 +29,43 @@ locality_zoneid = $FDB_ZONE_ID
 
 Operator will replicate pods across nodes using a *preferred* pod anti-affinity rule (`preferredDuringSchedulingIgnoredDuringExecution`) which discourages running multiple pods of the same process class on the same node, for any given cluster.
 
+### fault domain from node label
+
+**NOTE**: This feature requires the [unified image](customization.md#unified-vs-split-images).
+
+The `fdb-kubernetes-monitor` has support to read the labels from the node where it's running on an populate those as environment variables.
+The node label keys will be prefixed by `"NODE_LABEL"` and "/" and "." will be replaced in the key as "_". E.g. from the label `"foundationdb.org/testing = awesome"` the env variables
+`NODE_LABEL_FOUNDATIONDB_ORG_TESTING = awesome"` will be generated. This feature requires that the according RBAC permissions for the unified image (`ServiceAccount`) are setup.
+Those labels can then be added as localities or used as `ValueFrom` in the `FaultDomain` configuration of the cluster config.
+The environment variable name in `ValueFrom` must be prefixed with a `$`.
+
+## Unified vs Split Images
+
+The operator currently supports two different image types: a split image and a unified image.
```

Comment:
this section feels like it should just be a link to the `customization.md#unified-vs-split-images` (or be skipped since it is linked above too) as it looks like duplicated text from there that is likely to change within 6months